bin/test_sequencer

```diff
@@ -135,4 +135,10 @@ diff -u out/generated.md docs/specs/sequences/generated.md || (
 echo Checking for duplication in out/sequencer.out and etc/sequencer_planning.txt
 bin/test_sequencer_plan
 
+echo Checking that the sequences in validator/sequences are correct
+ls -1 sites/udmi_site_model/out/devices/AHU-1/tests/ \
```

Comment:
My concern here is that this would ultimately end up being too unstable, since it would be checking all the alpha/etc... tests.  Also, we should just make sure this is reconciled with the bit in test_sequencer that checks the generated.md file, since it's functionally supposed to be the same thing.